README.md

```diff
@@ -36,6 +36,7 @@ in any environment before the Docker container is started or tests are run.
 | ---| --- |
```

Comment:
The branch name and description don't meet practise.

Name: ticketNumber/short-description
Description: ticket link

Reply:
Just knowing about using ticket link as description. Please is there any literature I can reference in that regard? Many thanks.